### DISCUSSION TITLE
ModuleNotFoundError: No module named 'airflow_local_settings'

### DISCUSSION THREAD

**Discussion by seblum** [apache/airflow#32528] [Q&A]
Hi,

I am running the airflow community helm chart on an AWS EKS cluster using the KubernetesExecutor. While running the DAG  seems fine I cannot access remote logging, even though the Helm chart is configured correctly. 

After some digging I found that I run into an `Unable to load custom logging, using default config instead` within my scheduler or webserver. 

<img width="1348" alt="Bildschirmfoto 2023-07-11 um 18 39 17" src="https://github.com/apache/airflow/assets/64776588/7835d16b-36a7-4db0-91f6-49da68ce8c34">

I am running a custom airflow image and I assume its related to this. Updating it to the current version 2.6.3 did not fix the issue.

```
ARG AIRFLOW_VERSION=2.6.3

FROM apache/airflow:2.6.3-python3.11
USER airflow

COPY requirements.txt /
RUN pip install --no-cache-dir "apache-airflow==${AIRFLOW_VERSION}" -r /requirements.txt
```

I am not sure where the issue might be. I would assume the `airflow_local_settings` module is provided by the airflow image itself. Any help would be appreciated

**Comment by potiuk** [comment]
This is not an error - this is debug message. You can define your own `airflow_local_settings.py` in your PYTHONPATH to override local settings. You need to look further.

**Reply by seblum** [reply]
I am aware that I can define my own `airflow_local_settings.py`. Yet I thought there is like a default one.

Thanks for your reply! I will look further into it.
